static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -245,7 +247,7 @@ export function importSchema(
     profiles: input.profiles.map(profile =>
       importSingleProfile(profile, frameIndex, {
         ...options,
-        profileIds: input.shared.profile_ids,
+        profileIds: input.shared.profile_ids ?? input.shared.profiles,
```

Comment:
This will infer the reference from either of the two fields and store it on the frame obj under a single type